src/OrgUsageDashboard.vue

```diff
@@ -70,11 +70,17 @@ export default {
           this.isLoading = false;
           this.showDatePicker = true;
         });
+    },
+    generateDates: function() {
+      let startDate = moment().add(-1, 'month').format('YYYY-MM-DD');
+      let endDate = moment().format('YYYY-MM-DD');
+
+      return {startDate, endDate};
```

Comment:
@upplabs-alex-levchenko Spaces within `{ }` if it's a one-liner.

```suggestion
      return { startDate, endDate };
```

Reply:
Fixed